infra/relayer/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@origin/relayer",
+  "description": "Origin relayer service",
+  "version": "0.1.0",
+  "engines": {
+    "node": "10.x"
+  },
+  "author": "Origin Protocol Inc",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/OriginProtocol/origin/issues"
+  },
+  "dependencies": {
```

Comment:
A lot of these don't look to be necessary? I don't see any database so all `pg`, `sequelize`, `sequelize-cli` can go. Are `validator` or `url` used?

Reply:
Good catch, removed validator and url.
I'll keep the sequelize and pg one since we'll need them soon for DB access.